lib/generators/blueprinter/blueprint_generator.rb

```diff
@@ -65,7 +65,7 @@ def identifier_symbol
 
       def fields
         fs = if options["detect_fields"]
-               [].concat(options["fields"], introspected_fields)
+               Array.new(options["fields"]).concat(introspected_fields)
```

Comment:
What happens if `--fields` and `--detect_fields` overlap with the same field names? There would be duplicates right?
Why don't we use something like  `Array.new(options["fields"]) | introspected_fields`

Reply:
the array gets ```uniq``` called on it; I added a test for this